File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/CompanyMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Company;
import pl.yalgrin.playnite.simplesync.library.dto.CompanyDTO;

@Component
public class CompanyMapper extends AbstractObjectMapper<Company, CompanyDTO> {

    @Override
    protected CompanyDTO createDTO() {
        return new CompanyDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/CompanyMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Company
import pl.yalgrin.playnite.simplesync.library.dto.CompanyDTO

@Component
class CompanyMapper : LibraryObjectMapperImpl<Company, CompanyDTO>() {
    override fun createDTO(): CompanyDTO = CompanyDTO()
}